server/mqtt.go

```diff
@@ -217,6 +225,20 @@ type mqttAccountSessionManager struct {
 	replicas   int
 	rrmLastSeq uint64        // Restore retained messages expected last sequence
 	rrmDoneCh  chan struct{} // To notify the caller that all retained messages have been loaded
+	sp         sessPersist   // Used for cluster-wide processing of session records being persisted
+	domainTk   string        // Domain (with trailing "."), or possibly empty. This is added to session subject.
```

Comment:
If empty should it be "_."?

Reply:
I am not sure I need to make a special token for empty. What I do when needed is simply to trim the prefix `$MQTT.sess.` + this value, which if empty is fine.

In other words, I don't think I need the `$MQTT.sess.` subject prefix to include something special if there is no domain.

Reply:
Gotcha